### ISSUE TITLE
Error with latest Skungee in a bungeecord 1.16 setup

### ISSUE THREAD

**Issue by ChoqueCastroLD** [Skungee/Skungee#180]
When joining to a server
```
16:08:01 [INFO] [Shoko] <-> ServerConnector [lobby] has connected
16:08:01 [SEVERE] java.lang.NoSuchMethodException: me.limeglass.skungee.bungeecord.protocol.ProtocolPlayer.setDimension(java.lang.Number)
16:08:01 [SEVERE]     at java.base/java.lang.Class.getMethod(Class.java:2108)
16:08:01 [SEVERE]     at me.limeglass.skungee.bungeecord.protocol.LoginPacketHandler.handle(LoginPacketHandler.java:31)
16:08:01 [SEVERE]     at net.md_5.bungee.protocol.packet.Login.handle(Login.java:181)
16:08:01 [SEVERE]     at me.limeglass.skungee.bungeecord.protocol.channel.ChannelHandler.write(ChannelHandler.java:25)
16:08:01 [SEVERE]     at io.netty.channel.AbstractChannelHandlerContext.invokeWrite0(AbstractChannelHandlerContext.java:717)
16:08:01 [SEVERE]     at io.netty.channel.AbstractChannelHandlerContext.invokeWriteAndFlush(AbstractChannelHandlerContext.java:764)
16:08:01 [SEVERE]     at io.netty.channel.AbstractChannelHandlerContext.write(AbstractChannelHandlerContext.java:790)
16:08:01 [SEVERE]     at io.netty.channel.AbstractChannelHandlerContext.writeAndFlush(AbstractChannelHandlerContext.java:758)
16:08:01 [SEVERE]     at io.netty.channel.DefaultChannelPipeline.writeAndFlush(DefaultChannelPipeline.java:1020)
16:08:01 [SEVERE]     at io.netty.channel.AbstractChannel.writeAndFlush(AbstractChannel.java:299)
16:08:01 [SEVERE]     at net.md_5.bungee.netty.ChannelWrapper.write(ChannelWrapper.java:60)
16:08:01 [SEVERE]     at net.md_5.bungee.UserConnection$1.sendPacket(UserConnection.java:145)
16:08:01 [SEVERE]     at net.md_5.bungee.ServerConnector.handle(ServerConnector.java:218)
16:08:01 [SEVERE]     at net.md_5.bungee.protocol.packet.Login.handle(Login.java:181)
16:08:01 [SEVERE]     at net.md_5.bungee.netty.HandlerBoss.channelRead(HandlerBoss.java:105)
16:08:01 [SEVERE]     at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379)
16:08:01 [SEVERE]     at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365)
16:08:01 [SEVERE]     at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:357)
16:08:01 [SEVERE]     at io.netty.handler.codec.MessageToMessageDecoder.channelRead(MessageToMessageDecoder.java:103)
16:08:01 [SEVERE]     at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379)
16:08:01 [SEVERE]     at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365)
16:08:01 [SEVERE]     at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:357)
16:08:01 [SEVERE]     at io.netty.handler.codec.MessageToMessageDecoder.channelRead(MessageToMessageDecoder.java:103)
16:08:01 [SEVERE]     at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379)
16:08:01 [SEVERE]     at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365)
16:08:01 [SEVERE]     at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:357)
16:08:01 [SEVERE]     at io.netty.handler.codec.ByteToMessageDecoder.fireChannelRead(ByteToMessageDecoder.java:324)
16:08:01 [SEVERE]     at io.netty.handler.codec.ByteToMessageDecoder.channelRead(ByteToMessageDecoder.java:296)
16:08:01 [SEVERE]     at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379)
16:08:01 [SEVERE]     at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365)
16:08:01 [SEVERE]     at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:357)
16:08:01 [SEVERE]     at io.netty.handler.timeout.IdleStateHandler.channelRead(IdleStateHandler.java:286)
16:08:01 [SEVERE]     at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379)
16:08:01 [SEVERE]     at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365)
16:08:01 [SEVERE]     at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:357)
16:08:01 [SEVERE]     at io.netty.channel.DefaultChannelPipeline$HeadContext.channelRead(DefaultChannelPipeline.java:1410)
16:08:01 [SEVERE]     at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379)
16:08:01 [SEVERE]     at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365)
16:08:01 [SEVERE]     at io.netty.channel.DefaultChannelPipeline.fireChannelRead(DefaultChannelPipeline.java:919)
16:08:01 [SEVERE]     at io.netty.channel.epoll.AbstractEpollStreamChannel$EpollStreamUnsafe.epollInReady(AbstractEpollStreamChannel.java:792)
16:08:01 [SEVERE]     at io.netty.channel.epoll.EpollEventLoop.processReady(EpollEventLoop.java:475)
16:08:01 [SEVERE]     at io.netty.channel.epoll.EpollEventLoop.run(EpollEventLoop.java:378)
16:08:01 [SEVERE]     at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:989)
16:08:01 [SEVERE]     at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
16:08:01 [SEVERE]     at java.base/java.lang.Thread.run(Thread.java:834)
```

**Comment by TheLimeGlass** [comment]
Need your Proxy, Skungee, and Skript versions.

**Comment by BestInTest** [comment]
Same error.
Waterfall-358
Skript 2.4.1
Skungee 1.0.14

**Comment by ChoqueCastroLD** [comment]
![image](https://user-images.githubusercontent.com/46389454/86010827-8d711500-b9e1-11ea-9764-355b04a1a5d8.png)

Waterfall-358

**Comment by TheLimeGlass** [comment]
Try downloading 1.0.14 at https://github.com/Skungee/Skungee/releases/tag/1.0.14

**Comment by ChoqueCastroLD** [comment]
Still happening with 1.0.14 from https://github.com/Skungee/Skungee/releases/tag/1.0.14
Error from waterfall-358
```
[17:02:24 ERROR]: java.lang.NoSuchMethodException: me.limeglass.skungee.bungeecord.protocol.ProtocolPlayer.setDimension(java.lang.Number)
[17:02:24 ERROR]:       at java.base/java.lang.Class.getMethod(Class.java:2108)
[17:02:24 ERROR]:       at me.limeglass.skungee.bungeecord.protocol.LoginPacketHandler.handle(LoginPacketHandler.java:31)
[17:02:24 ERROR]:       at net.md_5.bungee.protocol.packet.Login.handle(Login.java:181)
[17:02:24 ERROR]:       at me.limeglass.skungee.bungeecord.protocol.channel.ChannelHandler.write(ChannelHandler.java:25)
[17:02:24 ERROR]:       at io.netty.channel.AbstractChannelHandlerContext.invokeWrite0(AbstractChannelHandlerContext.java:717)
[17:02:24 ERROR]:       at io.netty.channel.AbstractChannelHandlerContext.invokeWriteAndFlush(AbstractChannelHandlerContext.java:764)
[17:02:24 ERROR]:       at io.netty.channel.AbstractChannelHandlerContext.write(AbstractChannelHandlerContext.java:790)
[17:02:24 ERROR]:       at io.netty.channel.AbstractChannelHandlerContext.writeAndFlush(AbstractChannelHandlerContext.java:758)
[17:02:24 ERROR]:       at io.netty.channel.DefaultChannelPipeline.writeAndFlush(DefaultChannelPipeline.java:1020)
[17:02:24 ERROR]:       at io.netty.channel.AbstractChannel.writeAndFlush(AbstractChannel.java:299)
[17:02:24 ERROR]:       at net.md_5.bungee.netty.ChannelWrapper.write(ChannelWrapper.java:60)
[17:02:24 ERROR]:       at net.md_5.bungee.UserConnection$1.sendPacket(UserConnection.java:152)
[17:02:24 ERROR]:       at net.md_5.bungee.ServerConnector.handle(ServerConnector.java:251)
[17:02:24 ERROR]:       at net.md_5.bungee.protocol.packet.Login.handle(Login.java:181)
[17:02:24 ERROR]:       at net.md_5.bungee.netty.HandlerBoss.channelRead(HandlerBoss.java:105)
[17:02:24 ERROR]:       at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379)
[17:02:24 ERROR]:       at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365)
[17:02:24 ERROR]:       at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:357)
[17:02:24 ERROR]:       at io.netty.handler.codec.MessageToMessageDecoder.channelRead(MessageToMessageDecoder.java:103)
[17:02:24 ERROR]:       at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379)
[17:02:24 ERROR]:       at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365)
[17:02:24 ERROR]:       at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:357)
[17:02:24 ERROR]:       at io.netty.handler.codec.MessageToMessageDecoder.channelRead(MessageToMessageDecoder.java:103)
[17:02:24 ERROR]:       at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379)
[17:02:24 ERROR]:       at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365)
[17:02:24 ERROR]:       at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:357)
[17:02:24 ERROR]:       at io.netty.handler.codec.ByteToMessageDecoder.fireChannelRead(ByteToMessageDecoder.java:324)
[17:02:24 ERROR]:       at io.netty.handler.codec.ByteToMessageDecoder.channelRead(ByteToMessageDecoder.java:296)
[17:02:24 ERROR]:       at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379)
[17:02:24 ERROR]:       at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365)
[17:02:24 ERROR]:       at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:357)
[17:02:24 ERROR]:       at io.netty.handler.timeout.IdleStateHandler.channelRead(IdleStateHandler.java:286)
[17:02:24 ERROR]:       at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379)
[17:02:24 ERROR]:       at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365)
[17:02:24 ERROR]:       at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:357)
[17:02:24 ERROR]:       at io.netty.channel.DefaultChannelPipeline$HeadContext.channelRead(DefaultChannelPipeline.java:1410)
[17:02:24 ERROR]:       at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379)
[17:02:24 ERROR]:       at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365)
[17:02:24 ERROR]:       at io.netty.channel.DefaultChannelPipeline.fireChannelRead(DefaultChannelPipeline.java:919)
[17:02:24 ERROR]:       at io.netty.channel.epoll.AbstractEpollStreamChannel$EpollStreamUnsafe.epollInReady(AbstractEpollStreamChannel.java:792)
[17:02:24 ERROR]:       at io.netty.channel.epoll.EpollEventLoop.processReady(EpollEventLoop.java:475)
[17:02:24 ERROR]:       at io.netty.channel.epoll.EpollEventLoop.run(EpollEventLoop.java:378)
[17:02:24 ERROR]:       at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:989)
[17:02:24 ERROR]:       at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
[17:02:24 ERROR]:       at java.base/java.lang.Thread.run(Thread.java:834)
```

**Comment by TheLimeGlass** [comment]
Fixed in 1.0.15
